controllers/grid/files/proof/ProofFilesGridHandler.inc.php

```diff
@@ -34,6 +34,14 @@ function ProofFilesGridHandler() {
 			SUBMISSION_EMAIL_PROOFREAD_NOTIFY_AUTHOR,
 			ASSOC_TYPE_PUBLICATION_FORMAT
 		);
+		$this->addRoleAssignment(
+			array(
+				ROLE_ID_SUB_EDITOR,
+				ROLE_ID_MANAGER,
+				ROLE_ID_ASSISTANT
```

Comment:
Again, this one just opens the modal to select the files, but if press assistants shouldn't be able to do that, then this needs to be fixed.